zdg/game.py

```diff
@@ -35,6 +35,7 @@ def build_move_counter(self):
         return MoveCounter(self.screen)
 
     def build_problem(self):
+        # return Problem(difficulty=self.window.difficulty)
```

Comment:
Did you mean to keep this comment in?

Reply:
Problem will have a parameter for difficulty, but it may look different so can remove

Reply:
no preference from me